components/tools/OmeroWeb/omeroweb/filesessionstore.py

```diff
@@ -110,7 +113,11 @@ def load(self):
                     session_data = {}
                     self.delete()
                     self.create()
+            else:
+                logger.debug("No file_data for session: %s" %
+                             self._key_to_file())
         except (IOError, SuspiciousOperation):
+            logger.debug(traceback.format_exc())
```

Comment:
The `exc_info` attribute is always available to the logger so there is no real need to import `traceback` and call `format_exc()` here.